js/state-of-form.js

```diff
@@ -0,0 +1,40 @@
+import {getRemoveClass, getAdClass} from './util.js';
+
+const elementClassDisabled = [
+  'ad-form',
+  'map__filters',
+];
+
+function activateForms() {
+  elementClassDisabled.forEach((className) => {
+    const block = document.querySelector(`.${className}`);
+
+    getRemoveClass(block, `${className}--disabled`);
+    block.querySelectorAll('fieldset').forEach((element) => {
+      element.disabled = true;
```

Comment:
Лучше использовать setAttribute
```suggestion
      Element.setAttribute('disabled', true);
```